hibernate-core/src/main/java/org/hibernate/dialect/MySQL5Dialect.java

```diff
@@ -9,17 +9,25 @@
 import java.sql.SQLException;
 import java.sql.Types;
 
+import org.hibernate.dialect.function.VarArgsSQLFunction;
 import org.hibernate.dialect.hint.IndexQueryHintHandler;
 import org.hibernate.exception.spi.TemplatedViolatedConstraintNameExtracter;
 import org.hibernate.exception.spi.ViolatedConstraintNameExtracter;
 import org.hibernate.internal.util.JdbcExceptionHelper;
+import org.hibernate.type.StandardBasicTypes;
 
 /**
  * An SQL dialect for MySQL 5.x specific features.
  *
  * @author Steve Ebersole
  */
 public class MySQL5Dialect extends MySQLDialect {
+
+	public MySQL5Dialect() {
+		super();
+		// bitwise functions
+		registerFunction( "bitand", new VarArgsSQLFunction( StandardBasicTypes.LONG, "(", "&", ")" ) );
+	}
```

Comment:
what about adding also `registerFunction( "bit_and", new StandardSQLFunction( "bit_and" ) );` ?
`

Reply:
Does MySQL even *have* a `bitand()`? According to the MySQL docs it's called `bit_and()`.